### PR TITLE
Added new files to firestore folder

### LINKED ISSUE #31: Google Cloud function that updates firestore on delete
**Context**: Adding an image file to the bucket gs://street2sat-uploaded triggers the street2sat prediction pipeline which adds a new row to the Firestore database with the object detection predictions from that image.

**Issue**: If an image file is deleted from the bucket gs://street2sat-uploaded the associated row of predictions for that image file remains in the Firestore database. This becomes an issue when image files have been moved (copied then deleted) which results in new duplicate prediction rows in the database.

**Potential solution**: Create a python Google Cloud function triggered on image file removal that would also remove the associated prediction row from the Firestore database.

### PR DESCRIPTION
**Pull request by Aniket-Parlikar**
Developed a Cloud Function that is triggered when an image file is deleted from the street2sat-uploaded bucket and then proceeds to delete the corresponding record from the Firestore DB.

### CODE REVIEW

File: scripts/firestore_delete/main.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#The given script creates a CLoud Function to that is triggered when an image file is deleted from the street2sat-uploaded bucket
+#and then proceeds to delete the corresponding record from the Firestore DB.
```
Comment (ivanzvonkov) [inline scripts/firestore_delete/main.py]: nit: Inline comments should start with space: https://peps.python.org/pep-0008/#inline-comments

File: scripts/firestore_delete/main.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    doc_check=doc.get()
+
+    #Checking if the document reference exists in the collections and deleting it.
+    if doc_check.exists:
+        logger.info("The record associated with the image file exists in the Firestore DB")
+        logger.info("Deleting the given document")
+        doc.delete()
+        logger.info("Document has been deleted.")
+    else:
+        logger.info('No such record associated with the image file exists in the Firestore DB!')
```
Comment (ivanzvonkov) [inline scripts/firestore_delete/main.py]: Some of the logs end with punctuation and some don't, best to be consistent

File: scripts/firestore_delete/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Function dependencies, for example:
+# package>=version
+google-cloud-firestore
```
Comment (ivanzvonkov) [inline scripts/firestore_delete/requirements.txt]: Is the package not available by default in the Cloud Functions environment?

**Comment by ivanzvonkov** [comment]
1. There's a `gcp` folder in this repository which is more fitting for this script
2. Could you add the line used to deploy this function to the deploy script? https://github.com/nasaharvest/street2sat/blob/main/deploy.sh
3. I think the function name: `delete-street2sat-prediction`  may be a little more obvious. What do you think?

Code looks good!

File: scripts/firestore_delete/requirements.txt (hunk 3 above)
Comment (Aniket-Parlikar) [reply scripts/firestore_delete/requirements.txt]: No it is needed to be mentioned in the requirements.txt file, else we get an error

**Comment by Aniket-Parlikar** [comment]
I have made the necessary changes as requested. Please check and let me know in case any additional changes are needed.

**Review by ivanzvonkov — APPROVED** [review]
LGTM! There are still some comments without a space after `#` but everything else is good. You can go ahead and merge.

**Pull request merged**